src/creator.html

```diff
@@ -20,12 +20,37 @@
 	</head>
 	<body ng-app='matching' ng-controller="matchingCreatorCtrl">
 		<div id='modal'></div>
+		<div ng-show= "questionBankDialog" class="overlay"></div>
```

Comment:
You should probably reuse the existing `modalbg` instead of inventing a second one just for the question bank dialog.